tests/index.spec.js

```diff
@@ -117,6 +130,19 @@ describe('ResizeObserver', () => {
     expect(instance.resizeObserver).toBeFalsy();
   });
 
+  it('orphan target should not trigger onResize', () => {
+    const onResize = jest.fn();
+    function Component(props) {
+      return <ResizeObserver onResize={props.onResize}>{!props.hide && <div />}</ResizeObserver>;
+    }
+    const wrapper = mount(<Component onResize={onResize} />);
+
+    wrapper.setProps({ hide: true });
+    wrapper.triggerResize();
+    jest.runAllTimers();
+    expect(onResize).not.toHaveBeenCalled();
+  });
```

Comment:
This test case can re-produce the removed target issue.

Reply:
In real scenario, assembling Tabs dynamically might raise this issue as well. Take https://yuque.com for example. When editing flowchart in yuque editor, selection multiple items by mouse (which changes selections drastically) causes tabs on the right to re-render, which in turn causes following error

<img width="1393" alt="image" src="https://user-images.githubusercontent.com/252317/90384363-015f9d80-e0b4-11ea-8d2e-402d6e53d675.png">